### PR TITLE
feat: add windows node exporter

### PR DESCRIPTION
**Pull request by jdongo-aneo**

### CODE REVIEW

File: infrastructure/quick-deploy/aws/parameters.tfvars [hunk 1]
```diff
@@ -359,6 +359,13 @@ node_exporter = {
   node_selector = {}
 }
 
+windows_exporter = {
+  node_selector = {
+    "service" = "windows"
```
Comment (lemaitre-aneo) [inline infrastructure/quick-deploy/aws/parameters.tfvars:364]: the label should most likely not be service but os or plateform.

Comment (jdongo-aneo) [reply infrastructure/quick-deploy/aws/parameters.tfvars:364]: yes, platform will be used

File: infrastructure/quick-deploy/aws/variables.tf [hunk 2]
```diff
@@ -823,3 +823,16 @@ variable "static" {
...
 }
 
+variable "windows_exporter" {
+  description = "Windows exporter configuration"
+  type = object({
+    image_name        = optional(string, "ghcr.io/prometheus-community/windows-exporter")
+    image_tag         = optional(string)
+    pull_secrets      = optional(string, "")
+    init_image_name   = optional(string, "mcr.microsoft.com/windows/nanoserver")
+    init_image_tag    = optional(string, "ltsc2022")
```
Comment (lemaitre-aneo) [inline infrastructure/quick-deploy/aws/variables.tf:833]: The default tag should be null, and use the one defined in versions.tfvars.json

File: versions.tfvars.json
Comment (lemaitre-aneo) [inline versions.tfvars.json:1]: You most likely need to update the infra to a new pre-release for this to work

File: infrastructure/quick-deploy/aws/variables.tf (hunk 2 above)
Comment (lemaitre-aneo) [reply infrastructure/quick-deploy/aws/variables.tf:833]: You should remove the default value for the optional

**Pull request merged**